internal/pkg/logproto/README

```diff
@@ -0,0 +1,6 @@
+In order to avoid pulling in a lot of problematic dependencies, the
+contents of this directory have been copied verbatim from
```

Comment:
we might want to add a script to update this dir to keep it in sync with upstream.

updates will become a routine operation to keep up with loki.

Reply:
Thinking about this ... not sure.

There are some small changes in Loki version 2, but they are related to the server side of things, not really the client.

In principle I would prefer to NOT provide a path for upgrading to a newer version without having to carefully consider what that change entails.